app/controllers/application_controller.rb

```diff
@@ -97,6 +98,15 @@ def disable_caching
     response.headers['Pragma'] = 'no-cache'
   end
 
+  def cache_issuer_in_cookie
+    return unless current_sp
+
+    cookies[:sp_issuer] = {
+      value: current_sp.issuer,
+      expires: Figaro.env.issuer_cookie_expiration,
```

Comment:
We should probably add a value for this to application.yml.default.

Reply:
I had a default of 2 hours, but then if they end their session it will stick around. I opted to go without an expiration, but left in the ability for us to configure one if we later think it is necessary.

Reply:
Gotcha